soh/src/code/z_en_item00.c

```diff
@@ -386,42 +429,175 @@ void EnItem00_Init(Actor* thisx, GlobalContext* globalCtx) {
             this->scale = 0.02f;
             break;
         case ITEM00_ARROWS_SINGLE:
-            yOffset = 400.0f;
-            Actor_SetScale(&this->actor, 0.02f);
-            this->scale = 0.02f;
-            break;
+			if (CVar_GetS32("gNewDrops", 0) !=0) {
+				Actor_SetScale(&this->actor, 0.2f);
+				this->scale = 0.2f;
+				yOffset = 50.0f;
+				shadowScale = 0.5f;
+				this->actor.world.rot.x = 0x4000;
+			} else {
+				yOffset = 400.0f;
+	            Actor_SetScale(&this->actor, 0.02f);
+	            this->scale = 0.02f;
+			}
+			break;
         case ITEM00_ARROWS_SMALL:
+			if (CVar_GetS32("gNewDrops", 0) !=0) {
+				Actor_SetScale(&this->actor, 0.2f);
+				this->scale = 0.2f;
+				yOffset = 50.0f;
+				shadowScale = 0.5f;
+				this->actor.world.rot.x = 0x4000;
+			} else {
+				yOffset = 400.0f;
+	            Actor_SetScale(&this->actor, 0.02f);
+	            this->scale = 0.02f;
```

Comment:
```suggestion
            Actor_SetScale(&this->actor, 0.035f);
            this->scale = 0.035f;
            yOffset = 250.0f;
```

Reply:
There is a way bigger issue than actor scale and value there, I rechecked my code and I totally fucked up the case logic.
I placed break where they should not be and it made it all messed. 
This fallback default value are not needed and should not be wrote if the break; was placed correctly.
I am actually remaking it correctly and checking every scenario possible.
Cause right now Hearts, the one from Inside Deku tree was is already in 3D in original game got all shrinked cause of this rescale.
(I am actually making a better code, I have yet no idea how to edit the PR to show the edited code only I will try myself, I am fairly new to Github that why the whole PR is a bit a mess sorry)